src/test/integration/util/common.ts

```diff
@@ -100,10 +111,7 @@
 	}
 
 	public async openFolder (folder: string): Promise<void> {
```

Comment:
<!--
  0 failure: 
  1 warning:  src/test/integrat...
  
  
  DangerID: danger-id-Danger;
  File: src/test/integration/util/common.ts;
  Line: 68;
-->

- :warning: src/test/integration/util/common.ts line 68 – Argument 'tiapp' should be typed with a non-any type. (@typescript-eslint/explicit-module-boundary-types)